index.html

```diff
@@ -14,13 +15,14 @@ <h1 id="logo">Aziz Mejri</h1>
       <button type="button" class="menu-bars">
         <i class="fas fa-bars"></i>
       </button>
-      <nav id="desktop-nav">
-        <ul>
+      <nav id="desktop-nav" class="hidden">
+        <ul class="menu pop-up hidden">
+          <li><button class= "cross">X</button></li>
           <li><a class="nav-link" href="#projects">Portoflio</a></li>
           <li><a class="nav-link" href="#about-me">About</a></li>
           <li>
             <a class="nav-link" href="#contact"
-              >Contact <i class="far fa-envelope"></i
+              >Contact <i class="far fa-envelope" id="env"></i
             ></a>
           </li>
         </ul>
```

Comment:
- [ ] As stated by the previous reviewer 😊, kindly consider adding the separated lines between each menu, take a look at the following table to get what I mean ⤵️
- [ ] Also consider using the same font weight and font size following the Figma design 💯

| Figma design | Your design| 
| --- | --- |
|  ![image](https://user-images.githubusercontent.com/83360636/211065305-30be01f4-1838-47f3-9b13-0c4ad67ae4fc.png) |  ![image](https://user-images.githubusercontent.com/83360636/211065376-53243fb2-6409-43d4-8650-7b7636c9208f.png) |